.gitlab-ci.yml

```diff
@@ -26,14 +26,14 @@ baseline-image:
      - echo $BRANCH
      - export COMMIT=${CI_COMMIT_SHA:0:7}
      - echo $COMMIT
-     - git clone https://github.com/$REPO/e2e-infrastructure.git
+     - git clone https://github.com/$BASELINE_REPO/e2e-infrastructure.git
```

Comment:
@yudaykiran can we put a comment over this about where the value of $BASELINE_REPO will be initialized ?

Reply:
@utkarshmani1997 Its mentioned in the commit message, its internal to gitlab.